src/lib/mongoBackend/mongoRegistrationDelete.cpp

```diff
@@ -67,7 +67,7 @@ void mongoRegistrationDelete
     return;
   }
 
-  reqSemTake(__FUNCTION__, "Mongo Delete Registration", SemReadOp, &reqSemTaken);
+  reqSemTake(__FUNCTION__, "Mongo Delete Registration", SemWriteOp, &reqSemTaken);
```

Comment:
This is not style fix, but a little bug. Delete operation must use write sem. @McMutton takes this into account in the future ;)

NTC (informative)